app/src/test/java/io/appium/uiautomator2/utils/w3c/W3CCapsUtilsTests.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.uiautomator2.utils.w3c;
+
+import io.appium.uiautomator2.common.exceptions.InvalidArgumentException;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class W3CCapsUtilsTests {
```

Comment:
How about adding a test case which has non-appium prefix caps like `"{\"goog:chromeOptions\":{\"args\":[\"--disable-popup-blocking\"]}}"`? It should be valid.

Reply:
yep, will do. In general, such caps should stay unchanged